editor/block-mover/mover-label.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Wordpress dependencies
+ */
+import { __, sprintf } from 'i18n';
+
+/**
+ * Return a label for the block movement controls depending on block position.
+ *
+ * @param  {number}  selectedCount Number of blocks selected.
+ * @param  {string}  type          Block type - in the case of a single block, should
+ *                                 define its 'type'. I.e. 'Text', 'Heading', 'Image' etc.
+ * @param  {number}  firstIndex    The index (position - 1) of the first block selected.
+ * @param  {boolean} isFirst       This is the first block.
+ * @param  {boolean} isLast        This is the last block.
+ * @param  {number}  dir           Direction of movement (> 0 is considered to be going
+ *                                 down, < 0 is up).
+ * @return {string}                Label for the block movement controls.
+ */
+export function getBlockMoverLabel( selectedCount, type, firstIndex, isFirst, isLast, dir ) {
+	const position = ( firstIndex + 1 );
+
+	if ( selectedCount > 1 ) {
+		return getMultiBlockMoverLabel( selectedCount, firstIndex, isFirst, isLast, dir );
+	}
+
+	if ( isFirst && isLast ) {
+		// translators: %s: Type of block (i.e. Text, Image etc)
```

Comment:
Looking at the output of `languages/gutenberg.pot` after running `npm run gettext-strings`, it appears these comments aren't being picked up. Running the code through [ASTExplorer](http://astexplorer.net/#/gist/d6fd6d8f9ec27a7312b135abfae973d8/5a1234076867231ab3657518682715d2e75f2208), it seems to be an issue that the comment isn't considered a `leadingComments` for the actual translation call. It can be fixed by the following:

```js
return sprintf( 
  /* translators: %s: Type of block (i.e. Text, Image etc) */
  __( 'Block "%s" is at the end of the content and can’t be moved down' ),
  type
);
```

However, I don't necessarily think you ought to need to make this change. I'd prefer if we could instead improve the tooling to be a bit more liberal in what it considers to be a leading comment for a translation, even if the actual function call occurs within `sprintf` or is the `return` value.